cmd/controller/app/server.go

```diff
@@ -63,31 +63,32 @@ func Run(s *ServerRunOptions) error {
 	config.Burst = s.ApiServerBurst
 	stopCh := server.SetupSignalHandler()
 
-	pgClient := pgclientset.NewForConfigOrDie(config)
+	schedClient := schedclientset.NewForConfigOrDie(config)
 	kubeClient := kubernetes.NewForConfigOrDie(config)
 
-	pgInformerFactory := pgformers.NewSharedInformerFactory(pgClient, 0)
-	pgInformer := pgInformerFactory.Scheduling().V1alpha1().PodGroups()
+	schedInformerFactory := schedformers.NewSharedInformerFactory(schedClient, 0)
+	pgInformer := schedInformerFactory.Scheduling().V1alpha1().PodGroups()
+	eqInformer := schedInformerFactory.Scheduling().V1alpha1().ElasticQuotas()
 
-	informerFactory := informers.NewSharedInformerFactoryWithOptions(kubeClient, 0, informers.WithTweakListOptions(func(opt *metav1.ListOptions) {
-		opt.LabelSelector = util.PodGroupLabel
```

Comment:
Because the pods that may be calculated into the elastic quota usage may not have this label.

Reply:
I see, this was used for instantiating PodGroup informer only.

Can you get rid of the With...Options:

```go
coreInformerFactory := informers.NewSharedInformerFactory(kubeClient, 0)
```

(as we have safety check logic in PodGroupController#podAdded(), so that should be good).

Reply:
Sorry, I overlooked it, I will fix it

Reply:
done